rtf_vis_tool/src/Components/Data_Association_PC.js

```diff
@@ -0,0 +1,113 @@
+import React, {useEffect, useState, useRef} from "react";
+import {Canvas, extend} from "react-three-fiber";
+import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
+import '../stylesheets/Data_Association_PC.css';
+
+//Loading Components
+import CoordinateGrid from './CoordinateGrid';
+import OrbitControlsComponent from './OrbitControlsComponent';
+import PointMesh from './PointMesh';
+
+//allows user to orbit and pan around the point cloud
+extend({OrbitControls})
+
+//Point Cloud Renderer Component
+//Spawned once the 'SfMData' node is clicked
+const Data_Association_PC = (props) => {
+    const [pointCloud, setPointCloud] = useState([]);
+    const [isCentered, setIsCentered] = useState(false);
+    const [showCoordGrid, setShowCoordGrid] = useState(true);
+    const lightgray = `rgb(163, 168, 165)`;
+    const pointSizeArr = [0.2,8,8];
+
+    //render points_3d from data association json file (passed in through props)
```

Comment:
can we switch to reading from the COLMAP points3D.txt file instead?

Reply:
yup. I see you merged your `start-colmap-export` branch into master. I'll pull that onto my branch, run `scene_optimizer.py`, and then try rendering the point cloud from bundle adjustment output